app/views/posts/new_form.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="form__container">
+<%= tag.h1('New Post', class: 'form__title') %>
+<%= form_with model: Post.new do |f| %>
+  <%= f.text_field :title, placeholder: 'Title' %>
+  <%= f.text_area :text, rows: 20, placeholder: 'Enter the content' %>
+  <%= f.submit 'Save' %>
+<% end %>
+<%= render 'layouts/alert', flash: flash %>
+</div>
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/169174940-2fa283d8-2fd7-43aa-8442-adbfa8970f28.png)

- [ ] Kindly note that `rails` is not finding a corresponding view(form) for creating a new post, this is because you name you named this file `new_form.html.erb` instead of `new.html.erb` :point_left:  is the default name rails uses.